README.md

```diff
@@ -27,8 +27,10 @@ The tool dispatches and runs trial jobs generated by tuning algorithms to search
 <table>
   <tbody>
     <tr align="center" valign="bottom">
+    <td>
+      </td>
       <td>
-        <b>Supported Frameworks</b>
+        <b>Supported</b>
```

Comment:
let's keep the "Supported Frameworks" there, "supported" is not better than the previous version.

Reply:
change to Frameworks&Libraries